.aiexclude

```diff
@@ -0,0 +1,2 @@
+fastlane/.env
+local.properties
```

Comment:
I wonder if we should also include all `Constants.kt` files since we usually have API keys there... NABD if they are leaked but might be worth adding just in case. The most important info lives on these files yeah.

Reply:
Good point! I added `Constants.kt` so any files with that name, now or in the future, will be excluded. I figured that's acceptable, but let me know if you rather have all current `Constants.kt` files specified by path.

Reply:
No, I think this is good 👍 Thanks!